### PR TITLE
docimports for services

### PR DESCRIPTION
**Pull request by goderbauer**
Part of https://github.com/flutter/flutter/issues/150800

### CODE REVIEW

**Review by LongCatIsLooong — APPROVED** [review]
> To get some numbers: Before we started there where 7,631 unresolved comment references in pkg:flutter and 8,260 in the entire repository.

Is there a command that I can use to see the list of unresolved references?

File: packages/flutter/lib/src/services/system_channels.dart [hunk 1]
```diff
@@ -2,6 +2,24 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+/// @docImport 'dart:typed_data';
```
Comment (srawlins) [inline packages/flutter/lib/src/services/system_channels.dart:5]: Genuine question: Is there anything to take-away from a file like this? So many necessary doc imports! Is it a smell? Is it just a sign of a well-documented library with lots of examples from not-imported code? I honestly didn't think this feature was needed _this_ much!

**Review by srawlins — APPROVED** [review]

**Comment by goderbauer** [comment]
> Is there a command that I can use to see the list of unresolved references?

Turn on the `comment_references` lint [here](https://github.com/flutter/flutter/blob/455be19d928fa8e0e5bb26904f2d0f57312794c6/analysis_options.yaml#L91) and run `flutter analyze --flutter-repo`. If you plan on fixing any of these lint me know so we can sync and don't duplicate work.

File: packages/flutter/lib/src/services/system_channels.dart (hunk 1 above)
Comment (goderbauer) [reply packages/flutter/lib/src/services/system_channels.dart:5]: This is a low-level glue file that connects the engine with the framework. It contains lots of hooks for a variety of mechanisms in higher levels. So, all the docs are "this is the hook to power the [foo] thing". Therefore, I am not surprised that this particular file has all these doc imports.

**Comment by matanlurey** [comment]
test-exempt: `@docImport` is code-comments only.

**Pull request merged**